java/src/main/java/org/rocksdb/Transaction.java

```diff
@@ -661,6 +661,17 @@ public List<byte[]> multiGetAsList(final ReadOptions readOptions, final List<byt
     return Arrays.asList(multiGet(nativeHandle_, readOptions.nativeHandle_, keysArray));
   }
 
+  public List<byte[]> multiGetAsList(final ReadOptions readOptions,
```

Comment:
Could you comment this method in the same style as the others in the class, and add a line of text in the comment to say that it uses the optimized path ?

Reply:
Sure, I added a comment.

Reply:
Thanks.